charts/haystack-api/pipelines/my_pipelines.yaml

```diff
@@ -0,0 +1,53 @@
+version: '1.1.0'
+
+components:    # define all the building-blocks for Pipeline
+  - name: DocumentStore
+    type: ElasticsearchDocumentStore
+    params:
+      host: localhost
+  - name: Retriever
+    type: ElasticsearchRetriever
```

Comment:
This should be `BM25Retriever` from haystack 1.4 on.

Reply:
@tstadel - Can I get an "default" example pipeline yaml? I am far from an expert when it comes to configuring pipelines in haystack.